### ISSUE TITLE
1.5 distortion and artefacts

### ISSUE THREAD

**Issue by Super-zapper** [TheLastBen/fast-stable-diffusion#1490]
First of all, I want to say thank you, @TheLastBen. That's great job you done

But I have some problems during training and spent some time without result trying to solve it and find solution, so I hope someone could say what am I doing wrong.

The thing is that when I train 1.5 or custom models based on 1.5 I get distorted results also with artefacts, but with 2.1 that looks pretty good with same settings. 

I used the most simple script and tried to do everything as default:
1) Run import google drive and then dependencies
2) Load model 1.5 with my huggingface token (by default)
3) Create new session (model version 1.5)
4) Load 30 instance images 512x512 with right names
![instance_images](https://user-images.githubusercontent.com/112967133/214877146-ab0b233f-b53c-46f6-82b8-b23da66a7564.png)
5) Captions and concept images untouched (as I got, that's only for huge amount of images)
6) Run training on default parameters
7) After train, run test for 1.5 and let's see what I get
First of all I run with empty prompt field and model tries to generate trained images. I thought, if you don't use activation word (name of your instance images) generations of this model would be almost not affected. Yeah, likely it's just overtrained, but that's not main problem. Even there you can see blue splatters on every image - that's artefacts I'm talking about. 
![blank_1-5](https://user-images.githubusercontent.com/112967133/214878647-2cd5bfc9-de8b-440c-9477-7504abb6d8f2.png)
Next, I tried prompt "girl" without activation word and you can see, that trained images definitely applied to every image (maybe overtrain, but that's not how I thought that works). And also you can see there blue splatter artefacts
![girl_1-5](https://user-images.githubusercontent.com/112967133/214882082-230f2739-7ad2-4c33-9241-93e8611f48b5.png)
And the most interesting and crazy part. I tried only activation word - name on instance images (vlrtmshn). Why it's so f'cking scary, she looks like 70 yo, damn. That's definitely not how that should work. 
![vlrtmshn_1-5](https://user-images.githubusercontent.com/112967133/214883332-10475764-8788-4569-8320-11db411130aa.png)
Well, you can say - that's just overtrain, but let's compare results with 2.1 model.
I made training absolutely same as for 1.5, but only changed model to 2.1-512
First - empty prompt. You can see, that trained pictures still applied, but much less and not a single blue splatter - good sign
![blank_2-1](https://user-images.githubusercontent.com/112967133/214884189-eb14fcb6-351a-44c5-8fdb-e8f9af73e3cb.png)
Second - girl. Same result - no artefacts, trained images applied, but much less than for 1.5
![girl_2-1](https://user-images.githubusercontent.com/112967133/214885936-b6ea19ef-d603-4a89-ab7f-d74bee9cab3c.png)
And the most interesting - activation word. And you can see - no artefacts and the most important - face looks almost same as on instance images. I think, if I'll play little bit with number of steps and learning rate, I will get perfect result.
![vlrtmshn_2-1](https://user-images.githubusercontent.com/112967133/214886251-f6075363-8ad5-4d3d-9c1e-594092e3f886.png)
So now I have few questions:
- Why on 1.5 face is distorted
- Why on 1.5 there appears blue artefacts
- Why on 1.5 trained image applied for any generation much more than at 2.1
- Why trained images applied for any generation for both models (maybe overtrain, but I want someone to confirm that)

Thanks for reading, hope someone could explain why that could happen.

**Comment by TheLastBen** [comment]
The issues you're getting in 1.5 is because of the learning rate, reduce it to 5e-6 to avoid them.

for the face appearing in every generation, that's overfitting and it's a common issue when finetuning on a single object

**Comment by Super-zapper** [comment]
Thanks, @TheLastBen. Now it's much better. Before I already tried to reduce learning rate but reduced too much. 
But if compare with 2.1 (second picture) something is still not absolutely right. Do you know, what can be changed also?
![vlrtmshn_1-5-5e-6](https://user-images.githubusercontent.com/112967133/215010905-f8925dcd-da86-42c6-b69b-568b57fcf30b.png)
![vlrtmshn_2-1](https://user-images.githubusercontent.com/112967133/215010978-c7fe2926-f8ae-4ce3-911c-7480dc730b2c.png)

**Comment by Super-zapper** [comment]
@2blackbar
Can you say, what learning rate and model do you use?

**Comment by Super-zapper** [comment]
well, at least I solved problem about blue splatters
I need to use VAE

**Comment by Super-zapper** [comment]
also, as I got, the problem of distorted faces is 1.5 model. It's just not so good with faces and distort them anyway
